buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -55,3 +57,7 @@ object Versions {
 val PluginDependenciesSpec.buildSrcVersions: PluginDependencySpec
     inline get() =
             id("de.fayard.buildSrcVersions").version(Versions.de_fayard_buildsrcversions_gradle_plugin)
+
+val PluginDependenciesSpec.detekt: PluginDependencySpec
+    inline get() =
```

Comment:
First time I see the `inline` + `get`  in the same usage. Nice!

Reply:
Haha thanks! Honestly, I just saw this in the previous line and did a quick search for usefulness. Turns out inline getters are quite handy, so I kept it.